model/otel/metrics.yaml

```
@@ -110,3 +110,98 @@ groups:
+    instrument: counter
+    unit: "{LogRecord}"
+
+  - id: metric.otel.sdk.processor.log.queue.size
+    type: metric
+    metric_name: otel.sdk.processor.log.queue.size
+    stability: development
+    brief: "The number of LogRecords in the queue of a given instance of an SDK log processor"
+    note: |
+      Only applies to LogRecord processors which use a queue, e.g. the SDK Batching LogRecord Processor.
+    instrument: updowncounter
+    unit: "{LogRecord}"
+    attributes:
+      - ref: otel.component.type
+      - ref: otel.component.name
+
+  - id: metric.otel.sdk.processor.log.queue.capacity
+    type: metric
+    metric_name: otel.sdk.processor.log.queue.capacity
+    stability: development
+    brief: "The maximum number of LogRecords the queue of a given instance of an SDK LogRecord processor can hold"
+    note: |
+      Only applies to LogRecord processors which use a queue, e.g. the SDK Batching LogRecord Processor.
+    instrument: updowncounter
+    unit: "{LogRecord}"
+    attributes:
+      - ref: otel.component.type
+      - ref: otel.component.name
+
+  - id: metric.otel.sdk.processor.log.processed_logs
+    type: metric
+    metric_name: otel.sdk.processor.log.processed_logs
+    stability: development
+    brief: "The number of LogRecords for which the processing has finished, either successful or failed"
+    note: |
+      For successful processing, `error.type` MUST NOT be set. For failed processing, `error.type` must contain the failure cause.
+      For the SDK Simple and Batching LogRecord Processor a LogRecord is considered to be processed already when it has been submitted to the exporter,
+      not when the corresponding export call has finished.
+    instrument: counter
+    unit: "{LogRecord}"
+    attributes:
+      - ref: otel.component.type
+      - ref: otel.component.name
+      - ref: error.type
+        brief: >
+          A low-cardinality description of the failure reason. SDK Batching LogRecord Processors MUST use `queue_full` for LogRecords dropped due to a full queue.
+        examples: ["queue_full"]
+
+  - id: metric.otel.sdk.exporter.log.inflight_logs
+    type: metric
+    metric_name: otel.sdk.exporter.log.inflight_logs
+    stability: development
+    brief: "The number of LogRecords which were passed to the exporter, but that have not been exported yet (neither successful, nor failed)"
+    note: |
+      For successful exports, `error.type` MUST NOT be set. For failed exports, `error.type` must contain the failure cause.
+    instrument: updowncounter
+    unit: "{LogRecord}"
+    attributes:
+      - ref: otel.component.type
+      - ref: otel.component.name
+      - ref: server.address
+        requirement_level:
+          recommended: when applicable
+      - ref: server.port
+        requirement_level:
+          recommended: when applicable
+
+  - id: metric.otel.sdk.exporter.log.exported_logs
+    type: metric
+    metric_name: otel.sdk.exporter.log.exported_logs
```

I think it might be ok to remove the `_logs` suffix on all the metrics, similar to `jvm.class.loaded` / `jvm.class.unloaded`

```suggestion
    metric_name: otel.sdk.exporter.log.exported
```

---

SGTM. I also noticed I forgot about `otel.sdk.log.count` and renamed it to `otel.sdk.log.created` in [07f45ec](https://github.com/open-telemetry/semantic-conventions/pull/1921/commits/07f45ecf589cb520162178d3e52298efa5e08aa3).